com.unity.netcode.gameobjects/Tests/Runtime/BaseMultiInstanceTest.cs

```diff
@@ -53,6 +56,51 @@ public virtual IEnumerator Teardown()
             yield return new WaitUntil(() => Time.frameCount >= nextFrameNumber);
         }
 
+        /// <summary>
+        /// We want to exclude the TestRunner scene on the host-server side so it won't try to tell clients to
+        /// synchronize to this scene when they connect
+        /// </summary>
+        private bool VerifySceneIsValidForClientsToLoad(int sceneIndex, string sceneName, LoadSceneMode loadSceneMode)
+        {
+            // exclude test runner scene
+            if (sceneName.StartsWith(k_FirstPartOfTestRunnerSceneName))
+            {
+                return false;
+            }
+            return true;
+        }
+
+        /// <summary>
+        /// This registers scene validation with the server to prevent the server from telling connecting
+        /// clients to synchronize (i.e. load) the test runner scene.  This will also register the test runner
+        /// scene and its handle for both client(s) and server-host.
+        /// </summary>
+        private void SceneManagerValidationAndTestRunnerInitialization(NetworkManager networkManager)
+        {
+            // If VerifySceneBeforeLoading is not already set, then go ahead and set it so the host/server
+            // will not try to synchronize clients to the TestRunner scene.  We only need to do this for the server.
+            if (networkManager.IsServer && networkManager.SceneManager.VerifySceneBeforeLoading == null)
+            {
+                networkManager.SceneManager.VerifySceneBeforeLoading = VerifySceneIsValidForClientsToLoad;
+                // If a unit/integration test does not handle this on their own, then Ignore the validation warning
+                networkManager.SceneManager.IgnoreSceneValidationWarning = true;
+            }
+
+            // Register the test runner scene so it will be able to synchronize NetworkObjects without logging a
+            // warning about using the currently active scene
+            var scene = SceneManager.GetActiveScene();
```

Comment:
I'm a bit confused about this sceneManager, which NetworkManager is it associated with?

Reply:
This is not a "networkManager.SceneManager" but is a "UnityEngine.SceneManagement.SceneManager".   Just doing a sanity check that the developer writing the integration test hasn't already changed the currently active scene... if so then I don't handle the rest.